README.md

```diff
@@ -1,37 +1,58 @@
 # uk analysis dashboard
 <!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
 [![All Contributors](https://img.shields.io/badge/all_contributors-7-orange.svg?style=flat-square)](#contributors-)
+[![ease of contribution: easy](https://img.shields.io/badge/ease%20of%20contribution:%20easy-32bd50)](https://github.com/openclimatefix#how-easy-is-it-to-get-involved) 
 <!-- ALL-CONTRIBUTORS-BADGE:END -->
```

Comment:
you might have to move to this outside the `<!-- ALL-CONTRIBUTORS-BADGE:END -->` bit